.env.dev.example

```diff
@@ -0,0 +1,8 @@
+# If deploying against a fork, pass the address of a large token holder.
+# This address will be impersonated to distribute tokens to addresses inside test cases.
+# The whale needs to hold at least 3000 tokens
+TEST_TOKEN_WHALE="0x0000000000000000000000000000000000000000"
+
+# If you are testing with FORK_TEST_MODE='fork-existing', define the address of the
```

Comment:
It's a bit confusing that this env var isn't defined in any .env file but I can see later where this gets injected.

Reply:
Fair point @jordaniza. Since the developer experience is about using the makefile (rather than tweaking it), we can maybe reference the related targets (`make test-exfork-testnet`, ...) instead of naming the env var?